src/core/services/compiler/compiler.js

```diff
@@ -124,7 +124,7 @@ function MdCompilerProvider($compileProvider) {
    * - `>=1.7` - the compiler calls the constructor first before assigning bindings and
    *    `$compileProvider.preAssignBindingsEnabled()` no longer exists.
    *
-   * The default value is `false` but will change to `true` in AngularJS Material 1.2.
+   * The default value is `false` in to AngularJS 1.6 and earlier but `true` in AngularJS 1.7.
```

Comment:
`in to AngularJS 1.6` -> `in AngularJS 1.6`

Reply:
Also, the info that the default will change to `true` (for all AngularJS versions) in Material 1.2 was useful as well, I think it should stay here.